src/prerendered.rs

```diff
@@ -114,19 +113,22 @@ where
         Ok(())
     }
 
-    fn send_data(&mut self) -> Result<(), E> {
-        if cfg!(feature = "mosi_idle_high") {
-            for _ in 0..140 {
-                self.spi.write(from_ref(&0))?;
-            }
-        }
+    /// Add a reset sequence (140 zeroes) to the data buffer
+    fn flush(&mut self) -> Result<(), Error<E>> {
+        const FLUSH_DATA_LEN: usize = 140;
+        const FLUSH_DATA: &[u8] = &[0x00; FLUSH_DATA_LEN];
```

Comment:
Is it really necessary to include 140 `0` bytes in the data segment to load  zeroes into the slice? I think some kind of loop (or similar) is probably a better approach.